README.adoc

```diff
@@ -1,155 +1,173 @@
-|link:applications/sink/mqtt-sink/README.adoc[MQTT]
 |link:applications/processor/script-processor/README.adoc[Script]
+|link:applications/sink/mqtt-sink/README.adoc[MQTT]
 |link:applications/source/rabbit-source/README.adoc[RabbitMQ]
-|link:applications/sink/rabbit-sink/README.adoc[RabbitMQ]
 |link:applications/processor/splitter-processor/README.adoc[Splitter]
+|link:applications/sink/rabbit-sink/README.adoc[RabbitMQ]
 |link:applications/source/s3-source/README.adoc[AWS S3]
-|link:applications/sink/redis-sink/README.adoc[Redis]
 |link:applications/processor/transform-processor/README.adoc[Transform]
+|link:applications/sink/redis-sink/README.adoc[Redis]
 |link:applications/source/sftp-source/README.adoc[SFTP]
-|link:applications/sink/router-sink/README.adoc[Router]
 |link:applications/processor/twitter-trend-processor/README.adoc[Twitter Trend]
+|link:applications/sink/router-sink/README.adoc[Router]
 |link:applications/source/tcp-source/README.adoc[TCP]
-|link:applications/sink/sftp-sink/README.adoc[SFTP]
 |
+|link:applications/sink/sftp-sink/README.adoc[SFTP]
 |link:applications/source/time-source/README.adoc[Time]
-|link:applications/sink/tasklauncher-sink/README.adoc[Task Launcher]
 |
+|link:applications/sink/tasklauncher-sink/README.adoc[Task Launcher]
 |link:applications/source/twitter-message-source/README.adoc[Twitter Message]
-|link:applications/sink/tcp-sink/README.adoc[TCP]
 |
+|link:applications/sink/tcp-sink/README.adoc[TCP]
 |link:applications/source/twitter-search-source/README.adoc[Twitter Search]
-|link:applications/sink/throughput-sink/README.adoc[Throughput]
 |
+|link:applications/sink/throughput-sink/README.adoc[Throughput]
 |link:applications/source/twitter-stream-source/README.adoc[Twitter Stream]
-|link:applications/sink/twitter-message-sink/README.adoc[Twitter Message]
 |
+|link:applications/sink/twitter-message-sink/README.adoc[Twitter Message]
 |link:applications/source/websocket-source/README.adoc[Websocket]
-|link:applications/sink/twitter-update-sink/README.adoc[Twitter Update]
 |
-|link:applications/sink/wavefront-sink/README.adoc[Wavefront]
+|link:applications/sink/twitter-update-sink/README.adoc[Twitter Update]
 |
 |
+|link:applications/sink/wavefront-sink/README.adoc[Wavefront]
 |===
 
-=== Composite Function Support
+=== Composable Functions
 
-By default, the `source` applications are auto-configured with link:functions/function[functions] which may optionally be included in a composite function definition.
-This feature enables the sources to do things like
- - execute SpEL transformations
- - enrich message headers
- - filter events
- - produce task launch requests
+Spring Cloud Stream includes integration with Spring Cloud Function's function-based programming model that lets the
+business logic of an application be modeled as a `java.util.Supplier`, a `java.util.Function`, and a `java.util.Consumer`,
+representing the roles of a `Source`, a `Processor`, and a Sink, respectively.
```

Comment:
The `Source` and `Processor` are code, why `Sink` is not?

Reply:
`Sink` should definitely be marked as code.